tests/func/test_path_mapping.py

```diff
@@ -11,6 +11,50 @@
 from tests.helpers.timeline import Event
 
 
+def test_with_dot_remote_root(pyfile, tmpdir, run_as, start_method):
+    @pyfile
+    def code_to_debug():
+        from dbgimporter import import_and_enable_debugger
+        import_and_enable_debugger()
+        import os
+        import backchannel
+        backchannel.write_json(os.path.abspath(__file__))
+        print('done')
+
+    bp_line = 6
+    path_local = tmpdir.mkdir('local').join('code_to_debug.py').strpath
+    path_remote = tmpdir.mkdir('remote').join('code_to_debug.py').strpath
+
+    dir_local = os.path.dirname(path_local)
+    dir_remote = os.path.dirname(path_remote)
+
+    copyfile(code_to_debug, path_local)
+    copyfile(code_to_debug, path_remote)
+
+    with DebugSession() as session:
+        session.initialize(
+            target=(run_as, path_remote),
+            start_method=start_method,
+            ignore_unobserved=[Event('continued')],
+            use_backchannel=True,
+            path_mappings=[{
+                'localRoot': dir_local,
+                'remoteRoot': '.',
+            }],
+            cwd=dir_remote,
+        )
+        session.set_breakpoints(path_remote, [bp_line])
+        session.start_debugging()
+        hit = session.wait_for_thread_stopped('breakpoint')
+        frames = hit.stacktrace.body['stackFrames']
+        assert frames[0]['source']['path'] == Path(path_local)
```

Comment:
Can you also print `frames` here, for better diagnostics if the test fails in an automated run?